src/facetgroups/CheckboxFacetGroup.js

```diff
@@ -30,12 +34,17 @@ class CheckboxFacetGroup extends React.Component {
   }
 
   render() {
-    const {facetGroupName, facets} = this.props
+    const {facetGroupName, facetGroupNameDescription, facets} = this.props
     const {checkedFacets} = this.state
 
     return (
       <div className={`padding-bottom-xlarge`}>
-        <h4>{facetGroupName}</h4>
+        <h4>{facetGroupName}
+        {facetGroupNameDescription != null &&
```

Comment:
You can simplify this to just `facetGroupNameDescription &&`, as the [official React doc examples](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator).